src/test/java/org/cactoos/iterator/ImmutableTest.java

```diff
@@ -67,9 +71,17 @@ public void decoratesHasNext() {
         final Iterator<Integer> immutable = new Immutable<>(
             new IteratorOf<>(value)
         );
-        MatcherAssert.assertThat(immutable.hasNext(), new IsEqual<>(true));
+        new Assertion<>(
+            "hasNext is false for not traversed iterator",
```

Comment:
@ccostin93 Can you use "Must ..." here too?

Reply:
fixed